Wrappers/Python/cil/io/NikonDataReader.py

```diff
@@ -77,30 +77,38 @@ def __init__(self,
         
         self.file_name = kwargs.get('file_name', None)
         self.roi = kwargs.get('roi', {'angle': -1, 'horizontal': -1, 'vertical': -1})
-        self.normalize = kwargs.get('normalize', False)
+        self.normalise = kwargs.get('normalise', True)
         self.mode = kwargs.get('mode', 'bin')
         self.fliplr = kwargs.get('fliplr', False)
         
+        if kwargs.get('normalize', None):
```

Comment:
I think it should be 
```python
if 'normalize' in kwargs.keys():
    ...
```

Reply:
However, I think that the logic isn't right. Shouldn't we allow both ways but just not at once?

Reply:
Yes to the if statement change.

However the logic change complicates things in set-up where normalise has a default value. We don't know if they passed in both, or just `normalize`. I feel like it's a bit of messy code to accept one but not both, but just saying it's deprecated and crashing gives the relevant information.